modules/virt-creating-vm-template.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-vm-template_{context}"]
+= Creating a virtual machine from a template
+
+You can create a virtual machine (VM) from a template with an available boot source by using the {product-title} web console.
+
+.Procedure
+
+. Click *Virtualization* -> *Catalog* in the side menu.
+
+. Click *Boot source available* to filter templates with boot sources.
++
+The catalog displays the default templates. Click *All Items* to view all available templates for your filters.
+
+. Click a template tile to view its details.
+
+. Click *Quick create VirtualMachine* to create a VM from the template.
++
+Optional: Update parameters such as the VM name or disk source:
+
+.. Click *Customize VirtualMachine*.
+.. Edit the parameters and click *Next*.
+.. Click *Create VirtualMachine*.
```

Comment:
Nit: `.. Click *Create VirtualMachine*.` seems like it should be a top-level step (Step 5 instead of 4c)

Reply:
Different button! You would think so, but no, the wonderful UX team created a separate workflow for "Quick create" and "Customize".